dpti/ti.py

```diff
@@ -919,6 +920,36 @@ def compute_task(job, inte_method, Eo, Eo_err, To, scheme="simpson"):
     return info
 
 
+def run_task(task_name, machine_file):
+    task_dir_list = glob.glob(os.path.join(task_name, "task.*"))
+    task_dir_list = sorted(task_dir_list)
+    work_base_dir = os.path.join(os.getcwd())
+    with open(machine_file) as f:
+        mdata = json.load(f)
+    machine = Machine.load_from_dict(mdata["machine"])
+    resources = Resources.load_from_dict(mdata["resources"])
+
+    submission = Submission(
+        work_base=work_base_dir,
+        resources=resources,
+        machine=machine,
+    )
+
+    task_list = [
+        Task(
+            command="ln -s ../../graph.pb graph.pb; lmp -in in.lammps",
+            task_work_path=ii,
+            forward_files=["in.lammps", "*.lmp"],
+            backward_files=["log*", "out.lmp", "traj.dump"],
+        )
+        for ii in task_dir_list
+    ]
```

Comment:
**Suggestion:** In the `run_task` function, the `task_list` is created with a list comprehension that includes a `Task` object. The `Task` object's `command` attribute is a hardcoded string. It would be better to make this command string a parameter of the `run_task` function to make the function more flexible and reusable. [enhancement]
```suggestion
    def run_task(task_name, machine_file, command):
        ...
        task_list = [
            Task(
                command=command,
                task_work_path=ii,
                forward_files=["in.lammps", "*.lmp"],
                backward_files=["log*", "out.lmp", "traj.dump"],
            )
            for ii in task_dir_list
        ]
        ...
```